lib/pure/asyncmacro.nim

```diff
@@ -118,7 +118,8 @@ proc getFutureVarIdents(params: NimNode): seq[NimNode] {.compileTime.} =
   for i in 1 ..< len(params):
     expectKind(params[i], nnkIdentDefs)
     if params[i][1].kind == nnkBracketExpr and
-       params[i][1][0].eqIdent("futurevar"):
+       params[i][1][0].eqIdent("FutureVar"):
```

Comment:
`FutureVar.astToStr` ?
more typesafe

Reply:
It's not the part of this PR.
And many `eqident`s are used in this file

Reply:
Sound useful!

Reply:
I will add this later in this PR.